pandas/core/indexes/base.py

```diff
@@ -2788,26 +2788,55 @@ def set_value(self, arr, key, value):
 
     def _get_level_values(self, level):
         """
+        Return an Index of values for requested level.
+
         Return an Index of values for requested level, equal to the length
         of the index.
 
         Parameters
         ----------
         level : int or str
-            ``level`` is either the integer position of the level in the
-            MultiIndex, or the name of the level.
+            It is either the integer position of the level in the
+            MultiIndex, or the `name` of the level.
+            If given as an integer, it must be between 0 and the
+            number of levels.
 
         Returns
         -------
         values : Index
-            ``self``, as there is only one level in the Index.
+            `self`, as there is only one level in the Index.
 
         See also
         ---------
         pandas.MultiIndex.get_level_values : get values for a level of a
                                              MultiIndex
-        """
 
+        Notes
+        ---------
+        For `Index`, level should be 0, since there are no multiple levels.
+
+        Examples
+        ---------
+
+        Create an Index:
+
+        >>> idx = pd.Index(list('abc'))
+
```

Comment:
I wouldn't bother show this for an Index.

Reply:
Please elaborate what kind of change you would like to us to make. MultiIndex already mentions the other example. In our view, working on an Index is also useful to see for users somewhere, and this would be a proper place for it.